task.py

```diff
@@ -11,13 +11,14 @@
 import host
 
 class Task(ABC):
-    def __init__(self, tft: TestConfig, index: int, node_name: str, tenant: bool):
+    def __init__(self, tft: TestConfig, index: int, node_name: str, tenant: bool, reuse_node: bool = False):
```

Comment:
Can we get rid of this?

Reply:
Tell me what you think about this: https://github.com/wizhaoredhat/ocp-traffic-flow-tests/pull/12#discussion_r1345082034